modules/AlphaWalletFoundation/AlphaWalletFoundation/CoinTicker/Types/TickerIdFilter.swift

```diff
@@ -73,6 +73,11 @@ public class TickerIdFilter {
     }
 
     func filterMathesInPlatforms(token: TokenMappedToTicker, tickerId object: TickerIdObject) -> Bool {
+        func isMatchingSymbolAndName(token: TokenMappedToTicker, tickerIdObject object: TickerIdObject) -> Bool {
```

Comment:
This seems to be old code?

Reply:
@oa-s  this is a bit different, (might move in another pr), updated code to return it back because i wasn't able to resolve ticker id for currencies like ether or other native cryptos